package/MDAnalysis/lib/formats/xdrlib.pyx

```diff
@@ -490,7 +490,7 @@ cdef class TRRFile(_XDRFile):
             The arguments do not match with previous saved frames.
         """
         if self.mode != 'w' :
-            raise RuntimeError('File opened in mode: {}. Writing only allow '
+            raise IOError('File opened in mode: {}. Writing only allow '
```

Comment:
Yea sorry, should have paid attention to this PR... Then maybe I'll pull request to your branch, instead, and conflicts become sorted out there? I also tried to standardize error messages.